### PR TITLE
Home page light theme implemented

### PR DESCRIPTION
**Pull request by agarcia-caicedo**
## Issue This PR Addresses
#1173, this is the first step towards completely changing Telescope's Theme, it only touches the components used in the Home Page and Theme.js

## Type of Change

- [ ] **Bugfix**: Change which fixes an issue
- [ ] **New Feature**: Change which adds functionality
- [ ] **Documentation Update**: Change which improves documentation
- [x] **UI**: Change which improves UI

## Description

The home page has been changed to match the new light theme design
image
image
image

## Checklist

- [ ] **Quality**: This PR builds and passes our npm test and works locally
- [ ] **Tests**: This PR includes thorough tests or an explanation of why it does not
- [x] **Screenshots**: This PR includes screenshots or GIFs of the changes made or an explanation of why it does not (if applicable)
- [ ] **Documentation**: This PR includes updated/added documentation to user exposed functionality or configuration variables are added/changed or an explanation of why it does not(if applicable)

### CODE REVIEW

File: src/frontend/src/components/Banner/Banner.jsx
Comment (cindyledev) [inline src/frontend/src/components/Banner/Banner.jsx:40]: If our intention is to add a theme, we shouldn't be hardcoding in hex codes for the colours. I see that you added this hex code in `theme.js`. Instead of `color: '#e5e5e5'`, you can do `color: theme.palette.background.default`, this will apply the colour from `theme.js`

File: src/frontend/src/components/Banner/Banner.jsx
Comment (cindyledev) [inline src/frontend/src/components/Banner/Banner.jsx:70]: Same here `color: theme.palette.background.default`

File: src/frontend/src/components/Banner/Banner.jsx
Comment (cindyledev) [inline src/frontend/src/components/Banner/Banner.jsx:100]: `color: theme.palette.background.default`

File: src/frontend/src/components/Banner/Banner.jsx
Comment (cindyledev) [inline src/frontend/src/components/Banner/Banner.jsx:13]: If this colour is repeated in our theme, we should add this to `theme.js`

File: src/frontend/src/components/Header/MobileHeader.jsx
Comment (cindyledev) [inline src/frontend/src/components/Header/MobileHeader.jsx:47]: `backgroundColor: theme.palette.primary.main`

File: src/frontend/src/components/Post/Post.jsx
Comment (cindyledev) [inline src/frontend/src/components/Post/Post.jsx:17]: `theme.palettle.secondary.main`

**Review by cindyledev — CHANGES_REQUESTED** [review]
Just a few nits. I didn't get all the hex codes but you get what I mean 🍡

File: src/frontend/src/components/Header/MobileHeader.jsx
Comment (agarcia-caicedo) [reply src/frontend/src/components/Header/MobileHeader.jsx:47]: I think I had issues with this one, where it wouldn't accept the theme, but I can try again. It's possible I had the wrong syntax

**Comment by humphd** [comment]
I love where this is going, and I'm *so* happy that you're iterating on this @agarcia-caicedo!

My feedback about the colour theme here is that the post headers are quite dark for a light theme.  I wonder if we can lighten that up somehow so it's not such a stark contrast with the post content.

**Comment by agarcia-caicedo** [comment]
@humphd, I softened the color, and made other changes to make the contrast work. This is the updated theme

image
image
image
image

**Review by cindyledev — APPROVED** [review]
Wow this is nice! I noticed the Search page and About page don't have the correct colours yet but we can fix that in follow up PRs. I'd like to get this merged ASAP so I can see the changes on dev.

**Review by manekenpix — APPROVED** [review]

File: src/frontend/src/components/Posts/Posts.jsx
Comment (humphd) [inline src/frontend/src/components/Posts/Posts.jsx:19]: What does the string `'primary'` mean vs. using the property `theme.palette.text.primary`?

**Review by humphd — APPROVED** [review]
This is excellent work.  I really like what you're doing here, and look forward to seeing all of it happen.

I have one question.  Confirm that you meant to do what you did in the spot I mention, or fix But the rest looks right to me.

As @cindyledev points out, we shouldn't ship this to prod until all the other pages get the same treatment.  I wonder if we should get a doc added in another PR that talks about how to use the theme and colour values in new code?

Thanks @agarcia-caicedo!

File: src/frontend/src/components/Posts/Posts.jsx
Comment (agarcia-caicedo) [reply src/frontend/src/components/Posts/Posts.jsx:19]: For that one, I made the changes before I knew I could use the theme.palette property. The 'primary' works in certain situations but I'm not sure which ones, so I'll change this for better consistency

**Comment by raygervais** [comment]
Echoing @cindyledev, before merge I think we should tackle about page since 90% of text is not visible. 
image

File: src/frontend/src/components/Posts/Posts.jsx
Comment (cindyledev) [reply src/frontend/src/components/Posts/Posts.jsx:19]: I'm wondering if `primary` is actually the default Material UI blue

**Comment by cindyledev** [comment]
> Echoing @cindyledev, before merge I think we should tackle about page since 90% of text is not visible.
> image

Shhhhhh Ray, I wanted to fix that! I need some easy PRs for my Hacktoberfest contributions

File: src/frontend/src/components/Posts/Posts.jsx
Comment (agarcia-caicedo) [reply src/frontend/src/components/Posts/Posts.jsx:19]: primary is whatever you set it up, I just don't know when you can just use 'primary' and when you need theme.palette

**Comment by agarcia-caicedo** [comment]
> 
> 
> Echoing @cindyledev, before merge I think we should tackle about page since 90% of text is not visible.
> image

I wanted to separate the pages, since it's a lot of files, but I think for the sake of a functioning website I should fix that

**Comment by humphd** [comment]
If we aren't going to push to staging, I'm OK with this happening in multiple PRs, and the site being less than perfect while that happens.

**Review by humphd — APPROVED** [review]

**Review by cindyledev — APPROVED** [review]

**Review by manekenpix — APPROVED** [review]

**Pull request merged**
